setup.py

```diff
@@ -131,6 +145,7 @@ def run(self):
 
 cmdclass['build'] = css_first(build)
 cmdclass['sdist'] = css_first(sdist)
+cmdclass['bdist_egg'] = bdist_egg if 'bdist_egg' in sys.argv else bdist_egg_disabled
```

Comment:
To paraphrase one of the PyPA folks, I was avoiding being "hostile to particular workflows". If it's important to someone to build an egg, it's fine by me, I just want to avoid the *implicit* egg caused by `setup.py install`. I figure there's no harm in the extra few characters to allow explicitly requested eggs.